.travis.yml

```diff
@@ -8,7 +8,7 @@ matrix:
     - os: linux
       compiler: clang
       sudo: required
-      env: CFLAGS="-g -Wall -O3 -fsanitize=address" LDFLAGS="-fsanitize=address"
+      env: CFLAGS="-g -Wall -O3 -fsanitize=address" LDFLAGS="-fsanitize=address -Wl,-rpath,`pwd`/inst/lib"
```

Comment:
Do you need to set rpath here as it's also being done on the configure line?

Reply:
Yes, it's needed due to `make -e` and I can't get rid of that because we want it for CFLAGS.  I can't get rid of the rpath stuff in the configure line as we need it for the non-clang builds.  It's an annoying duplication, but this is a working kludge.